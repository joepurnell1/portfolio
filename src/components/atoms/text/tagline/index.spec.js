import React from 'react';
import renderer from 'react-test-renderer';
import Tagline from './index';

describe('Tagline component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <Tagline>Return of the Mac, get em</Tagline>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });

  it('should render with custom colour as expected', () => {
    const render = renderer.create(
      <Tagline colour="#D06">What it is What it does What it is What it isn&apost</Tagline>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
