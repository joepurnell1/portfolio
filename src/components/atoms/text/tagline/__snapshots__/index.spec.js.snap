// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tagline component should render default as expected 1`] = `
<h4
  className="sc-bdVaJa iLbktY"
>
  Return of the Mac, get em
</h4>
`;

exports[`Tagline component should render with custom colour as expected 1`] = `
<h4
  className="sc-bdVaJa ePPtBs"
>
  What it is What it does What it is What it isn&apost
</h4>
`;
